velox/exec/tests/StreamingAggregationTest.cpp

```diff
@@ -385,22 +422,17 @@ TEST_F(StreamingAggregationTest, sortedAggregations) {
 }
 
 TEST_F(StreamingAggregationTest, distinctAggregations) {
-  auto data = makeRowVector({
-      makeFlatVector<int64_t>(1'000, [](auto row) { return row / 4; }),
-      makeFlatVector<int64_t>(1'000, [](auto row) { return row % 3; }),
-  });
+  auto size = 128;
```

Comment:
Why 123 and not say 1'000?

Reply:
128 & 1024 are the same scenario as they are all less or equal to the default output batch (1024, `testDistinctAggregation(keys)`) and greater than the smaller output batch (32, `testDistinctAggregation(keys, 32)`). So I used a smaller size the make the test run faster.